NCsvPerf.Test/NCsvPerf.Test.csproj

```diff
@@ -8,6 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.3" />
+    <PackageReference Include="TxtCsvHelper" Version="1.2.8" />
```

Comment:
This is not needed since this reference will come in transitively from NCsvPerf.csproj.